app/views/application/_navigation.html.slim

```diff
@@ -19,6 +19,8 @@ nav.navbar.navbar-top
           li= link_to t('volunteers'), volunteers_path
           - if policy(Department).index?
             li= link_to t('departments'), departments_path
+          - if current_user.department?
```

Comment:
This checks if the user has a department and if the user is a department_manager. But yea, maybe this should be in the policy. But I'm just not sure how... I'll find out.